README.md

```diff
@@ -1,6 +1,5 @@
 # runc
 
-[![Build Status](https://travis-ci.org/opencontainers/runc.svg?branch=master)](https://travis-ci.org/opencontainers/runc)
```

Comment:
Can we replace with a CI status for GitHub Actions? I see containerd has one in their readme; https://github.com/containerd/containerd

Reply:
Can be done in a follow up :)